prac_05/hex_colours.py

```diff
@@ -0,0 +1,12 @@
+COLOUR_TO_CODE = {"absolute zero": "#0048ba", "acid green": "#b0bf1a", "aliceblue": "#f0f8ff",
+                  "alizarin crimson": "#e32636", "amaranth": "#e52b50",
+                  "amber": "#ffbf00", "amethyst": "#9966cc", "antiqueWhite": "#faebd7", "antiquewhite1": "#ffefdb",
+                  "antiquewhite2": "#eedfcc"}
+
+colour_name = input("Colour name: ").lower()
```

Comment:
Do we need to be specific with our naming here? For the context of this program, would `name` be a acceptable variable name?